parser_test.go

```diff
@@ -832,6 +832,45 @@ func TestParseRule(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "flowbits",
+			rule: `alert http $HOME_NET any -> $EXTERNAL_NET any (msg:"ACTIVEX ATTACK Citrix Access Gateway Plug-in ActiveX Code Execution - SET"; flow:to_server,established; content:"epaq"; http_uri; flowbits:set,ETPRO.Citrix_AGAX; flowbits:noalert; classtype:attempted-user; sid:2803611; rev:4;)`,
```

Comment:
This appears to be an emerging-threats (ET PRO even based on the flowbit) rule based on the SID. I'd like to keep third party rules out of this test file. Please make this a generic rule that exercises the necessary components of the code without relying on third party (especially paid) detections.